script.program.homeassistant/addon.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.program.homeassistant" name="Home Assistant" version="1.1.0" provider-name="LaTrappe">
+	<requires>
+		<import addon="script.module.requests" version="2.18.0"/>
+		<import addon="script.module.iso8601" version="0.1.12"/>
+		<import addon="script.module.yaml" version="4.1"/>
+		<import addon="xbmc.python" version="3.0.0"/>
+	</requires>
+	<extension point="xbmc.python.pluginsource" library="addon.py">
+		<provides>executable</provides>
+	</extension>
+	<extension point="xbmc.addon.metadata">
+		<platform>all</platform>
+		<summary lang="en_GB">Home Assistant Kodi Addon</summary>
+		<description lang="en_GB">This is a unofficial Home Assistant Kodi Addon (HAKA!)</description>
+		<summary lang="nl_NL">Home Assistant Kodi Addon</summary>
+		<description lang="nl_NL">Dit is een onofficiele Home Assistant Kodi Addon (HAKA!)</description>
+		<license>GNU General Public License, v2</license>
+		<language>en nl</language>
+		<source>https://github.com/LaTrappe/HAKA</source>
+		<website>https://www.home-assistant.io/</website>
+		<email>samsonjasper@gmail.com</email>
+		<assets>
+			<icon>icon.png</icon>
+			<fanart>fanart.jpg</fanart>
+			<screenshot>resources/screenshots/screenshot-01.jpg</screenshot>
+			<screenshot>resources/screenshots/screenshot-02.jpg</screenshot>
+			<screenshot>resources/screenshots/screenshot-03.jpg</screenshot>
+			<screenshot>resources/screenshots/screenshot-04.jpg</screenshot>			
+		</assets>
+		<news>
```

Comment:
Please keep the news tag only for the most recent version. Use the changelog.txt for a complete changelog. 
Reasoning: this is generated on the repo and served to every kodi installation. If we can make the addons.xml.gz as smaller as possible...the better.